pootle/apps/accounts/management/commands/purge_user.py

```diff
@@ -11,9 +11,8 @@
 
 
 class Command(UserCommand):
-    args = "user"
     help = "Delete user and all related objects"
 
-    def handle(self, *args, **kwargs):
-        super(Command, self).handle(*args, **kwargs)
-        self.get_user(args[0]).delete(purge=True)
+    def handle(self, **options):
+        for user in options['user']:
```

Comment:
@julen yes indeed they did change, ironically to do what I think the code originally intended.  I've adjusted the docs as needed.  ``argparse`` already makes the needed changes for the commands ``--help``.